app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Comment < ApplicationRecord
+  validates :c_body, presence: true, length: { minimum: 10 }
+  validates :user_id, presence: true
+  validates :post_id, presence: true
```

Comment:
No need to valdate ids here, since it is an association field, it would automatically be field in, always

Reply:
well noted.....